core/examples/luigi-sample-angular/src/app/project/project.component.html

```diff
@@ -1,76 +1,97 @@
+			</li>
+			<li class="fd-list-group__item">
+				<a href="javascript:void(0)" (click)="luigiClient.linkManager().fromContext('project').navigate('/settings')">parent
+					by name: project to settings</a>
+				<app-code-snippet data="luigiClient.linkManager().fromContext('project').navigate('/settings')"></app-code-snippet>
+			</li>
+			<li class="fd-list-group__item">
+				<a href="javascript:void(0)" (click)="luigiClient.linkManager().fromClosestContext().withParams({foo: 'bar'}).navigate('settings')">project
+					to settings with params (foo=bar)</a>
+				<app-code-snippet data="luigiClient.linkManager().fromClosestContext().withParams({foo: 'bar'}).navigate('settings')"></app-code-snippet>
+			</li>
+			<li class="fd-list-group__item">
+				<a href="javascript:void(0)" (click)="luigiClient.linkManager().fromContext('FOOMARK').navigate('/settings')">parent
+					by name: with nonexisting context</a> (look at the console)
+				<app-code-snippet data="luigiClient.linkManager().fromContext('FOOMARK').navigate('/settings')"></app-code-snippet>
+			</li>
+			<li class="fd-list-group__item">
+				<a href="javascript:void(0)" (click)="luigiClient.linkManager().navigate('/settings', null, true)">with preserved view:
+					project to global settings and back</a>
+				<app-code-snippet data="luigiClient.linkManager().navigate('/settings', null, true)"></app-code-snippet>
+			</li>
+			<li class="fd-list-group__item check-path">
+				<span>Check if path exists</span>
+				<app-code-snippet data="luigiClient.linkManager().pathExists('{{pathExists.formValue}}')"></app-code-snippet>
+				<span>
+					<input type="text" [(ngModel)]="pathExists.formValue" (input)="resetPathExistsResult()" />
+					<button class="fd-button" (click)="checkIfPathExists()">Check</button>
+				</span>
+				<p class="check-path-result">
+					<ng-container *ngIf="pathExists.result === true">
+						Path {{pathExists.formValue}} exists!
+					</ng-container>
+					<ng-container *ngIf="pathExists.result === false">
+						Path {{pathExists.formValue}} does not exist!
+					</ng-container>
+				</p>
+			</li>
+		</ul>
+	</div>
 </section>
 
-<app-modal [modalActive]="modalActive" (modalClosed)="toggleModal()"></app-modal>
+<section class="fd-section" *ngIf="projectId && projectId !== 'overview'">
+	<div class="fd-panel">
+		<p><strong>LuigiClient - wrong paths in linkManager.navigate():</strong></p>
+		<ul class="fd-list-group">
+			<li class="fd-list-group__item">
+				<a href="javascript:void(0)" (click)="luigiClient.linkManager().navigate('/overview/maskopatol')">Partitialy wrong link</a>
+				<app-code-snippet data="luigiClient.linkManager().navigate('/overview/maskopatol')"></app-code-snippet>
+			</li>
+			<li class="fd-list-group__item">
+				<a href="javascript:void(0)" (click)="luigiClient.linkManager().navigate('/maskopatol/has/a/child')">Totally wrong link</a>
```

Comment:
When I click on it the first time, it shows the error message, which is good. Then I close this message, and click on it the second time, the error message is not shown again. 
If we would have more 'broken' links, this message would be shown only for the first one, not all of them, and it's not how it should work.